src/app/public/modules/clipboard/index.ts

```diff
@@ -1,3 +0,0 @@
-export * from './clipboard.module';
```

Comment:
Unfortunately, we can't delete any `index.ts` files once they've been released, since some applications may be importing from a long path, such as:
```
import { SkyClipboardService } from '@blackbaud/skyux-lib-clipboard/modules/clipboard';
```

Reply:
Ah, that's upsetting, but understandable.